Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:buster
+
+WORKDIR /dhx
+
+RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev openssl git clang libclang-dev curl vim unzip screen bash\
+	&& wget -O - https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2020-02-17 \
+	&& curl https://sh.rustup.rs -sSf | sh -s -- -y \
+	&& echo 'PATH="$/root/.cargo/bin:$PATH";' >> ~/.bash_profile && . ~/.bash_profile && . /root/.cargo/env \
```

Comment:
does this work with the `$` in front? or should it be `/root/.cargo/bin:$PATH`

Reply:
I have simplified this bit now